### DISCUSSION TITLE
会开mlc课程么

### DISCUSSION THREAD

**Discussion by ticoAg** [datawhalechina/DOPMC#93] [建议反馈]
目前没看到有相关的立项，机器学习部署作为ml应用一个重要环节，正在逐渐为人重视
tvm，llvm, MLIR，cuda，xla
gpu tpu npu fpga 各种软硬件生态 
onnx tensorRt部署
天奇大佬夏天开了mlc课程  可以借鉴学习 感觉差了亿点点tvm基础
其他的相关内容太多 感觉无从下嘴 
从c++开始么

**Comment by Sm1les** [comment]
同学你好，你有兴趣么？要不来试试？

**Reply by ticoAg** [reply]
我也只才开始学了一点点 开课准备教程这种还是找大佬吧🤔

**Reply by Sm1les** [reply]
好的，我这就开始招募
